pkg/operator/operator.go

```diff
@@ -121,23 +124,23 @@ func NewOperator() (context.Context, *Operator) {
 	})
 
 	// Client Config
-	config := controllerruntime.GetConfigOrDie()
+	config := ctrl.GetConfigOrDie()
 	config.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(float32(options.FromContext(ctx).KubeClientQPS), options.FromContext(ctx).KubeClientBurst)
 	config.UserAgent = fmt.Sprintf("%s/%s", appName, Version)
 
 	// Client
 	kubernetesInterface := kubernetes.NewForConfigOrDie(config)
 
 	// Logging
-	logger := logging.NewLogger(ctx, component)
-	ctx = knativelogging.WithLogger(ctx, logger)
+	logger := log.FromContext(ctx).WithName(component)
+	ctx = logr.NewContext(ctx, logger)
 	logging.ConfigureGlobalLoggers(ctx)
 
-	knativelogging.FromContext(ctx).With("version", Version).Debugf("discovered karpenter version")
+	log.FromContext(ctx).WithValues("version", Version).Info("discovered karpenter version")
 
 	// Manager
-	mgrOpts := controllerruntime.Options{
-		Logger:                        logging.IgnoreDebugEvents(zapr.NewLogger(logger.Desugar())),
+	mgrOpts := ctrl.Options{
+		Logger:                        logger,
```

Comment:
AFAIK, this changes our behavior, since we now see Debug events that come from the manager startup. How different does this make our logs, does it make it more spammy or are the logs reasonable?

Reply:
I didn't see us undo this change. Did we mean to resolve this comment since it looks like we still will fire debug events from the manager during startup which we didn't used to do before because we threw them out